server/services/database/users.js

```diff
@@ -82,9 +82,21 @@ const getUserById = id => {
   });
 };
 
+const getUserLanguages = email => {
```

Comment:
can you use the user id instead? email is not unique on our database

Reply:
done